Language/Functions/External Interrupts/attachInterrupt.adoc

```diff
@@ -59,7 +59,7 @@ For more information on interrupts, see http://gammon.com.au/interrupts[Nick Gam
 === Syntax
 `attachInterrupt(digitalPinToInterrupt(pin), ISR, mode);` (recommended) +
 `attachInterrupt(interrupt, ISR, mode);` (not recommended) +
-`attachInterrupt(pin, ISR, mode);` (Not recommended. Arduino SAMD Boards, Uno WiFi Rev2, Due, 101 only)
+`attachInterrupt(pin, ISR, mode);` (Not recommended. This syntax is for use with Arduino SAMD Boards, Uno WiFi Rev2, Due, 101 only.)
```

Comment:
Hi per
I think this is still somewhat confusing. What do you think of replacing 
`Not recommended. This syntax is for use with Arduino SAMD Boards, Uno WiFi Rev2, Due, 101 only.`
for
`Not recommended. Additionally, this syntax only works on Arduino SAMD Boards, Uno WiFi Rev2, Due, and 101`?